crawl-ref/source/dat/descript/unrand.txt

```diff
@@ -768,3 +768,10 @@ Force Lance
 A plain iron lance, imbued with supernatural gravitas via the artifice of the
 mage Gell. It strikes with a force many times stronger than that imparted by
 the wielder.
+%%%%
+toga "Victory"
+
+An ancient artefact of Okawaru, these woolen robes were lost when a former
+champion failed a duel. Its defense and damage sometimes increase when the
+wearer emerges victorious from threatening combat -- but beware! Its bonuses
+reset to zero if the wearer reads any scrolls, or quaffs any potions.
```

Comment:
A minor style note is that this could maybe be reworded slightly to tend more towards flavour than mechanics (since the mechanics are explicitly spelled out in the DBRAND) - maybe just something like "Its enchantments increase in power when the wearer emerges victorious from threatening combat, but resorting to the use of scrolls or potions for assistance causes them to fade away again."

Reply:
Ah, I hadn't seen this comment while writing the latest commit. No worries, that's easy to adjust.